plugins/main/host-device/host-device.go

```diff
@@ -38,12 +39,17 @@ import (
 	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
 )
 
+const (
+	sysBusPci = "/sys/bus/pci/devices"
```

Comment:
PTAL on https://github.com/golang/go/wiki/CodeReviewComments#initialisms - that should be named as `sysBusPCI`

Reply:
fixed